go.mod

```diff
@@ -205,11 +207,11 @@ require (
 replace (
 	// Ref: https://forum.cosmos.network/t/ibc-security-advisory-dragonberry/7702
 	github.com/confio/ics23/go => github.com/confio/ics23/go v0.9.0
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.46.10
+	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.46.11
 	github.com/crypto-org-chain/cronos/memiavl => ./memiavl
 	github.com/crypto-org-chain/cronos/versiondb => ./versiondb
 	github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.19-deepcopy-jumptable
-	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.20.4-cronos
+	github.com/evmos/ethermint => github.com/yihuang/ethermint v0.20.2-cronos.0.20230314094949-f1f845d74751
```

Comment:
https://github.com/crypto-org-chain/ethermint/pull/227

Reply:
do we need update change log for this [diff](https://github.com/crypto-org-chain/ethermint/compare/v0.20.4-cronos...crypto-org-chain:ethermint:f1f845d74751)